### ISSUE TITLE
BUG: The app won't start after update

### ISSUE THREAD

**Issue by Excelsus999** [wAIfu-DEV/w-AI-fu_v2#39]
I used the auto updater and everything seems fine, until....

https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/149912072/51cbdab0-4646-48f8-af67-fd14952e2916

**Comment by wAIfu-DEV** [comment]
can you show me the console please?

**Comment by Atsukohirota** [comment]
![image](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/130300501/5327ab09-fb50-4789-833c-cece42f68a42)

**Comment by wAIfu-DEV** [comment]
yeah that's the bad kind of bugs, only shows up for certain people, working on it

**Comment by wAIfu-DEV** [comment]
can you guys tell me your python version?
just typing `python --version` in cmd should tell you the version

**Comment by Atsukohirota** [comment]
3.10.4

**Comment by Excelsus999** [comment]
3.10.0b4

**Comment by moo7md1** [comment]
i have the same problem my python version is 3.11.7

**Comment by Atsukohirota** [comment]
be patient they are working on it

**Comment by Excelsus999** [comment]
![uuh2](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/149912072/03c435ec-25df-4bbf-8b62-116ded7ed96c)
![uuh](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/149912072/965f9204-0921-4c52-b345-cef2a69c4109)

**Comment by xxandeer** [comment]
The reason it's not working is because Git is not installed on the computer.
When running the new version, it installs the following in the virtual environment (venv):

```python
vectordb2==0.1.9
websockets==11.0.3
git+https://github.com/vioshyvo/mrpt/

Since Git is not installed, it fails.

**Comment by wAIfu-DEV** [comment]
Link for git: [https://git-scm.com/download/win](https://git-scm.com/download/win)
Will fix that with a hotfix

**Comment by Atsukohirota** [comment]
![image](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/130300501/c5955564-f6f0-4bbc-a801-c5d7ccdf476a)
so far so good

**Comment by wAIfu-DEV** [comment]
Fixed in 2.0.5, thanks to xxander
